app/models/transaction_type.rb

```diff
@@ -19,16 +21,33 @@ class TransactionType < ApplicationRecord
     ]
   }.freeze
 
+  TRANSACTION_TYPES_WITH_HINT_TEXT = %i[benefits].freeze
+
   scope :debits, -> { where(operation: :debit) }
   scope :credits, -> { where(operation: :credit) }
 
   def self.populate
+    populate_records
+    populate_sort_order(:credit, 1000)
+    populate_sort_order(:debit, 2000)
+  end
```

Comment:
I wonder if it would be simpler just to modify populate to:
```ruby
NAMES.each_with_index do |(operation, names), op_index|
  names.each_with_index do |name, index|
    start_number = (op_index * 1000) + (index * 10)
    transaction_type = find_or_initialize_by(name: name, operation: operation)
    transaction_type.update! start_number: start_number
  end
end
```
Oh and `100` and `1000` are too close to each other. There are currently seven income types. If only three more are added, the last one will get the start number from the first debit entry.